### ISSUE TITLE
Jetpack: Remove reference to Priority Support for free sites

### ISSUE THREAD

**Issue by beaulebens** [Automattic/wp-calypso#21053]
With a Free Jetpack site, view the My Plans page, and see this card:

![screen shot 2017-12-21 at 3 05 51 pm](https://user-images.githubusercontent.com/108942/34276860-884498c4-e660-11e7-9590-bb6c9372a2d9.png)

Without a paid plan, Jetpack sites do not actually get priority support. They can use our documentation, post in the plugin forums, or they can contact support still (but do not receive priority responses). Suggest rewording this to something like:

---

**Find answers in our documentation**
**Need help?** Search our support site to find out more about how to make the most of your Jetpack site.

---

I suggest we just remove the "Ask a question" button.

cc @nicoleckohler for better wordsmithing skills. @annezazu for suggestion on if we'd like to still suggest they contact support here.

**Comment by annezazu** [comment]
+1 on better wordsmithing. Where did this "Ask a question" take you out of curiosity? We should still offer support for free users but shouldn't phrase it as priority or act like we are giving more support to them (this reads like a paid user for sure).

**Comment by beaulebens** [comment]
"Ask a question" currently takes users directly to the contact form on jetpack.com.

**Comment by nicoleckohler** [comment]
@beaulebens Could we keep the current display for paid users (and let them contact support) but change the free user block to something like what you've suggested?

**Comment by beaulebens** [comment]
Oh sorry, yes absolutely meant to keep this for paying users, but reword
for free ones.

**Comment by arunsathiya** [comment]
Also, please take a note of this. When I visit https://wordpress.com/help/contact and choose a self-hosted WordPress (WordPress.org) site from the **_Which site do you need help with?_** list, the email is automatically sent to Jetpack Happiness Engineers. 
This gives users a chance to directly speak to Jetpack customer support, without even encouraging them to search the Jetpack self-service portal or forums. I have misused this a few times myself, haha.

**Comment by kraftbj** [comment]
Noting that "contact support" links for JP in Calypso takes them straight to the contact form on Jetpack.com without requiring them to search for a doc first.
